src/custom/utils/signatures.ts

```diff
@@ -129,31 +117,35 @@ async function _signOrderCancellation(params: SingOrderCancellationParams): Prom
     signer,
   })
 
-  return signOrderCancellationGp(domain, orderId, signer, getSigningSchemeLibValue(signingScheme))
+  return import('@cowprotocol/contracts').then((m) =>
+    m.signOrderCancellation(domain, orderId, signer, getSigningSchemeLibValue(signingScheme))
+  )
 }
 
-type SigningResult = { signature: string; signingScheme: SigningScheme }
+type SigningResult = { signature: string; signingScheme: EcdsaSigningScheme }
 
 async function _signPayload(
   payload: any,
   signFn: typeof _signOrder | typeof _signOrderCancellation,
   signer: Signer,
   signingMethod: 'default' | 'v4' | 'int_v4' | 'v3' | 'eth_sign' = 'v4'
 ): Promise<SigningResult> {
-  const signingScheme = signingMethod === 'eth_sign' ? SigningScheme.ETHSIGN : SigningScheme.EIP712
```

Comment:
Didn't you move all of this to the sdk already?

Reply:
Yes: https://github.com/cowprotocol/cowswap/pull/2187